examples/03-ui-components/21-table-reordering-visualization/index.html

```
@@ -0,0 +1,14 @@
+<html lang="en">
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)